mentat/terminal/prompt_completer.py

```diff
@@ -67,7 +65,9 @@ def refresh_completions_for_file_path(self, file_path: Path):
         self.syntax_completions[file_path] = SyntaxCompletion(words=filtered_tokens)
 
     async def refresh_completions(self):
-        file_paths = list(self.session.code_context.files.keys())
+        code_context = CODE_CONTEXT.get()
+
+        file_paths = list(code_context.files.keys())
```

Comment:
```suggestion
        file_paths = list(CODE_CONTEXT.get().files.keys())
```
I see this pattern a few places in the code, makes sense to use it if there's only one use.

Reply:
I think I would prefer to as a rule get the contextvars at the start of the function; also, if we end up merging all of the contextvars, we'll have to get it at the start anyways.